### PR TITLE
survey view | rebranding 4

### PR DESCRIPTION
**Pull request by mhigbyflowers**
The only files to that belong to this PR are: 
survey-view.component.html
survey-view.component.css
survey-view.component.ts

### CODE REVIEW

File: public-ui/src/app/views/survey-view/survey-view.component.ts [hunk 1]
```diff
@@ -8,16 +8,18 @@ import {ISubscription} from 'rxjs/Subscription';
 import {DataBrowserService} from '../../../publicGenerated/api/dataBrowser.service';
 import {AchillesResult} from '../../../publicGenerated/model/achillesResult';
 import {QuestionConcept} from '../../../publicGenerated/model/questionConcept';
+import {QuestionConceptListResponse} from '../../../publicGenerated/model/questionConceptListResponse';
 import {SurveyModule} from '../../../publicGenerated/model/surveyModule';
```
Comment (dolbeew) [inline public-ui/src/app/views/survey-view/survey-view.component.ts]: Can combine all `publicGenerated` paths into a single import. Also, `AchillesResult` and `QuestionConceptListResponse` seem to be unused.

**Review by dolbeew — COMMENTED** [review]
I'm unable to view the surveys, I get this error when I click 'View Complete Survey':
`Error: Uncaught (in promise): Error: Cannot match any routes. URL Segment: 'quick-search/survey/1585855'`

File: public-ui/src/app/views/survey-view/survey-view.component.ts (hunk 1 above)
Comment (mhigbyflowers) [reply public-ui/src/app/views/survey-view/survey-view.component.ts]: are you seeing these from tslint? Mine must need to be upgraded.

Comment (mhigbyflowers) [reply public-ui/src/app/views/survey-view/survey-view.component.ts]: also the link from the quick-search works now.

File: public-ui/src/app/views/survey-view/survey-view.component.html [hunk 2]
```diff
@@ -1,224 +1,208 @@
...
+      <div class="stat-container">
+        <h2 *ngIf="!loading" class="secondary-display">
+          <span>{{ surveyResult.items.length }} </span>
+        </h2>
+        <span *ngIf="loading" class="spinner spinner-md"> </span>
+        <p class="info-text">
+          Questions
+        </p>
+      </div>
+      <div class="pdf-link">
```
Comment (dolbeew) [inline public-ui/src/app/views/survey-view/survey-view.component.html:36]: We might want to add some conditions to hide this link and other text while loading, it seems to overlap with the loading spinner.

**Review by dolbeew — APPROVED** [review]
Looks good, just left one comment.

File: public-ui/src/app/views/survey-view/survey-view.component.ts (hunk 1 above)
Comment (dolbeew) [reply public-ui/src/app/views/survey-view/survey-view.component.ts]: Not sure if it's from tslint or not, it was highlighted and gave a suggestion on hover.

**Pull request merged**